### PR TITLE
systemd-mount: add page; systemd-umount: add alias page

### PR DESCRIPTION
**Pull request by varadrane7**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-mount.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# systemd-mount
+
+> Establish and destroy transient mount or auto-mount points.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-mount.html>.
+
+- Mount a file system (image or block device) and automatically determine the mount point:
```
Comment (acuteenvy) [inline pages/linux/systemd-mount.md:6]:
```suggestion
- Mount a file system (image or block device) at `/run/media/system/LABEL` where LABEL is the filesystem label or the device name if there is no label:
```

File: pages/linux/systemd-mount.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create an automount point that mounts the actual file system at the time of first access:
+
+`systemd-mount --automount=yes {{path/to/file_or_device}}`
+
+- Unmount multiple mount points or devices:
+
+`systemd-mount --umount {{path/to/mount_point_or_device1}} {{path/to/mount_point_or_device2}}`
+
+- Mount a file system (image or block device) with specific file system type:
```
Comment (acuteenvy) [inline pages/linux/systemd-mount.md:30]:
```suggestion
- Mount a file system (image or block device) with a specific file system type:
```

File: pages/linux/systemd-mount.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Show a list of all local, known block devices with file systems that may be mounted:
+
+`systemd-mount --list`
+
+- Create an automount point that mounts the actual file system at the time of first access:
+
+`systemd-mount --automount=yes {{path/to/file_or_device}}`
+
+- Unmount multiple mount points or devices:
```
Comment (acuteenvy) [inline pages/linux/systemd-mount.md:26]:
```suggestion
- Unmount one or more devices:
```

File: pages/linux/systemd-mount.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+`systemd-mount {{path/to/file_or_device}}`
+
+- Mount a file system (image or block device) at a specific location:
+
+`systemd-mount {{path/to/file_or_device}} {{path/to/mount_point}}`
+
+- Unmount a specific mount point or device:
+
+`systemd-mount --umount {{path/to/mount_point_or_device}}`
+
```
Comment (acuteenvy) [inline pages/linux/systemd-mount.md:17]:
```suggestion
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**